templates/workflows/changelogger.yml

```diff
@@ -14,6 +14,7 @@ jobs:
   check-changelog:
     name:    Check changelog
     runs-on: ubuntu-latest
+    if: "!contains(github.event.pull_request.body, '[skip-changelog]')"
```

Comment:
I'm curious: Why not use a label instead of adding something to the body? Same question for the other skips.

Reply:
That is a bit of pain to manage in general. Cause there are no labels that are global.